libraries/AP_HAL_ChibiOS/hwdef/f405-MatekGPS/hwdef.dat

```diff
@@ -222,8 +222,8 @@ PC5 BATT2_VOLTAGE_SENS ADC1 SCALE(1)
 # define BOARD_RSSI_ANA_PIN 12
 # define HAL_DEFAULT_AIRSPEED_PIN 13
 
-define HAL_BATT_MONITOR_DEFAULT 4
-define HAL_BATT2_MONITOR_DEFAULT 4
+define HAL_BATT_MONITOR_DEFAULT 0
+define HAL_BATT2_MONITOR_DEFAULT 0
```

Comment:
Why is this default being removed?

Reply:
because it is primarily a GPS, and it was broadcasting battery info for an unconfigured battery